### PR TITLE
Fix log types for grpc speaker and server42

### PR DESCRIPTION
**Pull request by rozdy**

### CODE REVIEW

File: docker/logstash/etc/logstash/conf.d/logstash.conf.example [hunk 1]
```diff
@@ -1,45 +0,0 @@
-input {
```
Comment (timofei-durakov) [inline docker/logstash/etc/logstash/conf.d/logstash.conf.example:1]: why did you remove example?

Comment (rozdy) [reply docker/logstash/etc/logstash/conf.d/logstash.conf.example:1]: Because it isn't used any more. This config is generated by corresponding template.

Comment (timofei-durakov) [reply docker/logstash/etc/logstash/conf.d/logstash.conf.example:1]: but this one is an example file, not a config itself

Comment (rozdy) [reply docker/logstash/etc/logstash/conf.d/logstash.conf.example]: Ok, reverted file back
